modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts

```diff
@@ -175,6 +175,16 @@ const get = (dom: DOMUtils) => {
       }
     },
 
+    lang: {
+      inline: 'span',
+      clear_child_styles: true,
+      remove_similar: true,
+      attributes: {
+        'lang': '%value',
+        'data-mce-lang': (vars) => vars?.customValue || null
```

Comment:
Should this be this instead? That way we're not using a falsy check:
```suggestion
        'data-mce-lang': (vars) => vars?.customValue ?? null
```